setup.cfg

```diff
@@ -47,7 +47,7 @@ edit_on_github = True
 github_project = astropy/astrowidgets
 # install_requires should be formatted as a comma-separated list, e.g.:
 # install_requires = astropy, scipy, matplotlib
-install_requires = astropy, ginga, pillow, ipyevents, opencv-python
```

Comment:
Note: I had problem installing with `opencv-python` in the requirements even though I have `opencv` installed. So have to remove this one. Not sure how this didn't trip the CIs. :woman_shrugging:

Reply:
I'll try to explore in the next day or two. Could we keep `ginga` and `pillow`? At the moment they are needed to use the widget at all..

Reply:
OK I added them back in `setup.cfg`.